src/flatc.cpp

```diff
@@ -237,12 +248,20 @@ int main(int argc, const char *argv[]) {
         for (size_t i = 0; i < num_generators; ++i) {
           if (arg == generators[i].generator_opt_long ||
               (generators[i].generator_opt_short &&
-               arg == generators[i].generator_opt_short)) {
-            generator_enabled[i] = true;
+               arg == generators[i].generator_opt_short) ||
+                arg == "--grpc") {
             any_generator = true;
-            goto found;
+            if(arg == generators[i].generator_opt_long ||
+              (generators[i].generator_opt_short &&
+               arg == generators[i].generator_opt_short)) {
+              generator_enabled[i] = true;
+            }
+            if(arg == "--grpc") {
+              grpc_enabled = true;
+            }
           }
         }
+        goto found;
```

Comment:
the loop should go to `found` on the first match. The way you've written it the `Error` is never executed!